packages/p2p/src/components/modal-manager/modals/ad-visibility-error-modal/ad-visibility-error-modal.jsx

```diff
@@ -13,7 +13,7 @@ const AdVisibilityErrorModal = ({ error_code }) => {
     if (error_code === api_error_codes.AD_EXCEEDS_BALANCE) {
         return (
             <Modal
-                className='p2p-my-ads__modal-error'
+                className='my-ads__modal-error'
```

Comment:
shouldn't we separate the scss from my-ads for modals to here instead? I think we should have one modal with one scss file

Reply:
maybe then you can separate it out into a separate styles folder within the modals folder and have the scss file there for the ones that have similar styling